library/VTKExtensions/Rendering/vtkF3DRenderer.cxx

```diff
@@ -766,14 +775,15 @@ void vtkF3DRenderer::ConfigureHDRITexture()
     this->SetEnvironmentTexture(this->HDRITexture);
 
 #if VTK_VERSION_NUMBER >= VTK_VERSION_CHECK(9, 2, 20221220)
-    // Force modified on the spherical harmonics
-    // to avoid them updating themselves
-    // TODO add support for not providing a texture and still using IBL in VTK
-    // https://github.com/f3d-app/f3d/issues/935
+    // No cache support before that, not needed
+#if VTK_VERSION_NUMBER < VTK_VERSION_CHECK(9, 3, 20230830) // TODO date
```

Comment:
Can you merge the two ifs?

Reply:
no, this code is need between 20221220 and 20230830

Reply:
```cpp
#if VTK_VERSION_NUMBER >= VTK_VERSION_CHECK(9, 2, 20221220) && VTK_VERSION_NUMBER < VTK_VERSION_CHECK(9, 3, 20230830)
// ...
#endif
```